CHANGELOG.md

```diff
@@ -13,6 +13,12 @@
   line ending.
 * Add a new syntax `EscapeHtmlSyntax` to encode (`"`), (`<`), (`>`) and (`&`).
 * Add an option `caseSensitive` to `TextSyntax`.
+* Add a new public method `parse(String text)` for `Document`.
+* Add a new public method `parseLineList(List<Line> text)` for `Document`.
+* Add a new type: `Line`.
+* **Breaking change**: Change the `line` properties of type `String` to `Line`.
```

Comment:
one small nit: put all breaks at the top together!

Reply:
Done